typescript-timers/interval.ts

```diff
@@ -0,0 +1,16 @@
+const $countdownDisplay = document.getElementById('countdown-display');
+if (!$countdownDisplay) throw new Error('$countdownDisplay query has failed');
+
+let count = 4;
+
+$countdownDisplay.textContent = count.toString();
+
+const setIntervalId = setInterval(() => {
+  count--;
+  if (count > 0) {
+    $countdownDisplay.textContent = count as unknown as string;
+  } else {
```

Comment:
You do not need to use type assertion here. What type does `textContent` require? How can we convert something _to_ that type?